applications/adv_networking_bench/adv_networking_bench_default_rx.yaml

```diff
@@ -53,36 +53,36 @@ advanced_network:
       access:
         - local
       num_bufs: 30720
-      buf_size: 1064        
-    
-    interfaces:      
+      buf_size: 1064
+
+    interfaces:
     - name: data2
       address: 0005:03:00.0
       rx:
-        - queues:
-          - name: "Default"
-            id: 0
-            cpu_core: 7
-            batch_size: 10240
-            output_port: "bench_rx_out"
-            memory_regions: 
-              - "Default_RX_CPU"
-          - name: "Data"
+        queues:
+        - name: "Default"
+          id: 0
+          cpu_core: 7
+          batch_size: 10240
+          output_port: "bench_rx_out"
+          memory_regions:
+            - "Default_RX_CPU"
+        - name: "Data"
+          id: 1
+          cpu_core: 8
+          batch_size: 10240
+          output_port: "bench_rx_out"
+          memory_regions:
+            - "Data_RX_CPU"
+            - "Data_RX_GPU"
+        flows:
+        - name: "ADC Samples"
+          action:
+            type: queue
             id: 1
-            cpu_core: 8
-            batch_size: 10240
-            output_port: "bench_rx_out"
-            memory_regions: 
-              - "Data_RX_CPU"
-              - "Data_RX_GPU"
-          flows:
-            - name: "ADC Samples"
-              action:
-                type: queue
-                id: 1
-              match:
-                udp_src: 4096 #12288
-                udp_dst: 4096 #12288                          
+          match:
+            udp_src: 4096 #12288
+            udp_dst: 4096 #12288
 bench_rx:
   split_boundary: true
```

Comment:
They're obsolete only at the ANO config level (so rx/tx operator), since redundant with the list of memory regions that tell you if you're splitting or not (multiple vs one region)

However, with the current design, the pre/post-processors of packets need to also have that information, so unfortunately they have to duplicate metadata to match the ANO config currently, and these need to stay.

Reply:
Thanks. Planned for refactoring or to be documented as an interface requirement?

Reply:
which part? the post-processor needing to indicate it? Could be either depending on design we align with with Cliff